task_3/counter.h

```diff
@@ -0,0 +1,41 @@
+#include <iostream>
+#include <boost/bind.hpp>
+#include <boost/asio.hpp>
+#include <boost/thread/thread.hpp>
+#include <experimental/filesystem>
+#include <chrono>
+#include <queue>
+#include <fstream>
+using std::string;
+using namespace std::experimental::filesystem;
+
+class Counter {
+	
+public:
+	Counter(path dir, string outFile);
+	~Counter() = default;
+
+	struct Statistics {					// store statistic data
+		size_t linesTotal = 0;
+		size_t linesBlank = 0;
+		size_t linesComment = 0;
+		size_t linesSource = 0;
+	};
```

Comment:
Define statistic right after Statistics structure and move this to the beginning of the class under private.

Reply:
ok